File: 0201_0300/0283/0283.py
import re
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    s = list(filter(None, re.split("([A-Z][a-z]*)", f.readline().strip())))
    ans = "Yes"
    for t in s:
        if len(t) < 2 or len(t) > 4 or t[0].islower() or not t[1:].islower():
            ans = "No"
            break
    q.write(ans)

File: 0201_0300/0283/0283.c
#include <stdio.h>
#include <ctype.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int l = 1, ans;
	char s[100001], *t;

	fscanf(f, "%s", s);
	t = s;
	ans = isupper(*t);
	while (*++t && ans)
		if ((isupper(*t) && l < 2) || (islower(*t) && ++l > 4))
			ans = 0;
		else if (isupper(*t))
			l = 1;
	fprintf(q, ans && l > 1 ? "Yes" : "No");
	return 0;
}
